### PR TITLE
removed any elements on the page not in the backlog or implemented

### PR DESCRIPTION
**Pull request by nigelpain**
Went through the prototype Mat created and removed anything that wasn't in development, then went through and added elements in the backlog.

### CODE REVIEW

File: app/views/display-title.html [hunk 1]
```diff
@@ -10,15 +10,7 @@
 
   {{>includes/beta-banner}}
 
-  <div class="global-breadcrumb">
```
Comment (Demotive) [inline app/views/display-title.html:13]: I'd suggest leaving the breadcrumbs in there.

File: app/views/display-title.html [hunk 2]
```diff
@@ -27,7 +19,7 @@ <h1 class="heading-large first-heading">
     </h1>
 
     <p>
-      This title was last changed on <b>15 October 2012</b> at <b>11:21:18</b>
+      This register was last changed on <b>15 October 2012</b> at <b>11:21:18</b>
```
Comment (Demotive) [inline app/views/display-title.html]: Keep "title" - this page doesn't just show a "register"...

File: app/views/display-title.html [hunk 3]
```diff
@@ -53,7 +45,7 @@ <h1 class="heading-large first-heading">
     </div>
 
     <dl class="definition-tabular--b">
-      <dt>Owner(s)</dt>
+      <dt>Owner/Proprietor(s)</dt>
```
Comment (Demotive) [inline app/views/display-title.html]: Would suggest keeping just "owner" - but there's arguments around this.

File: app/views/display-title.html [hunk 4]
```diff
@@ -223,75 +135,16 @@ <h4 class="heading-small collapse-bottom">Scotland Company PLC</h4>
...
 
-        <small class="register-datestamp">15 October 2012</small>
-        <p>The price stated to have been paid on 3 October 2012 was &pound;188,000.</p>
-
       </section>
 
       <section id="charges-register" class="text">
         <h2 class="heading-large">Charges register</h2>
 
-        <details>
```
Comment (Demotive) [inline app/views/display-title.html:234]: Keep the summaries.

File: app/views/display-title.html [hunk 5]
```diff
@@ -84,95 +72,19 @@ <h1 class="heading-large first-heading">
...
-
-        <small class="register-datestamp">10 October 2012</small>
-        <p>The land edged red on the attached plan has the benefit of the rights granted by the Lease of the top flat dated 3 October 2012 referred to in the Charges Register.</p>
-
-      </section>
-
       <section id="proprietorship-register" class="text">
         <h2 class="heading-large">Proprietorship register</h2>
 
-        <details>
```
Comment (Demotive) [inline app/views/display-title.html:164]: Keep the summaries

**Comment by Demotive** [comment]
I've suggested keeping some bits that don't currently appear in the build - specifically the breadcrumbs and help summaries for each register within the title.

Also 2 comments around wording. Both of which will probably be controversial :D

**Comment by nigelpain** [comment]
Added all the elements mentioned above @Demotive if you don't mind having another look. Thanks.

**Comment by Demotive** [comment]
LGTM :+1:

**Pull request merged**